parser/src/parser/parser.js

```diff
@@ -1348,9 +1346,8 @@ class Parser extends Obj {
       default:
         return null;
     }
-
+    // eslint-disable-next-line no-constant-condition
     while (1) {
```

Comment:
You changed `while(1)` to while(true)` above, but the rest of these are unchanged -- did we want to swap them all?

Reply:
Oh, no. I was just just trying to make the ESLint error go away before I actually read what the problem was. I'll change it back to `while(1)` for consistency.